src/controllers/orders.ts

```diff
@@ -29,19 +28,15 @@ export const postOrders: RequestHandler = async (
         transaction_id: transactionId,
       });
     } else if (email) {
-      order = await Order.create({
+      order = await orderServices.createOrder({
         email: email,
         status: ORDER_STATUS.ACTIVE,
```

Comment:
Where `ORDER_STATUS` has been imported in this page?

Reply:
I added imports statement 👍